telco-core/configuration/reference-crs/required/scheduling/NROPSubscription.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: numaresources-operator
   namespace: openshift-numaresources
 spec:
-  channel: "4.14"
+  channel: {{ .spec.channel }}
   name: numaresources-operator
-  source: redhat-operators-disconnected
-  sourceNamespace: openshift-marketplace
+  source: {{ if .spec.source }} {{ .spec.source }} {{ else }} "redhat-operators-disconnected " {{ end }}
```

Comment:
is there a reason that this has a default but Sriov doesn't?


also this would be easier to read and I think should do the same thing. 
```suggestion
source: {{ or .spec.source "redhat-operators-disconnected }}
```